src/redux/reducers/index.js

```diff
@@ -0,0 +1,6 @@
+import { combineReducers } from "redux";
+import loginReducer from "./loginReducer";
+
+const rootReducers = combineReducers({ loginReducer });
+
+export default rootReducers;
```

Comment:
Take note @micahoriaso has a similar file but with a different name, please liaise with him and figure this out